decompiler/ObjectFile/ObjectFileDB.cpp

```diff
@@ -222,11 +220,11 @@ void ObjectFileDB::get_objs_from_dgo(const std::string& filename) {
       }
 
       if (chunk_size < MAX_CHUNK_SIZE) {
-        lzo_uint bytes_written;
-        auto lzo_rv =
-            lzo1x_decompress(compressed_reader.here(), chunk_size,
-                             decompressed_data.data() + output_offset, &bytes_written, nullptr);
-        assert(lzo_rv == LZO_E_OK);
+        std::size_t bytes_written = 0;
+        lzokay::EResult ok = lzokay::decompress(compressed_reader.here(), chunk_size,
```

Comment:
They swapped around some of the arguments, so I think you want:
```
        lzokay::EResult ok = lzokay::decompress(
            compressed_reader.here(), chunk_size, decompressed_data.data() + output_offset,
            decompressed_data.size() - output_offset, bytes_written);
```

The `decompressed_data.size() - output_offset` is how many bytes are left in the decompression output buffer, and `bytes_written` is actually an output (passed by reference).  With these changes it appears to work. Most of the advanced decompilation passes fail, but I think this is expected.